packages/opentelemetry-instrumentation/test/common/Instrumentation.test.ts

```diff
@@ -56,4 +64,27 @@ describe('BaseInstrumentation', () => {
       assert.strictEqual(called, true);
     });
   });
+
+  describe('config', () => {
+    it('should get config', () => {
+      const instrumentation: Instrumentation = new TestInstrumentation({
+        isActive: false,
+      });
+      const configuration =
+        instrumentation.getConfig() as TestInstrumentationConfig;
+      assert.notStrictEqual(configuration, null);
+      assert.strictEqual(configuration.isActive, false);
+    });
+
+    it('should modify config', () => {
```

Comment:
1. add describe block
```
describe('setConfig', ()=> {})
```

2. 
```suggestion
    it('should set a new config for instrumentation', () => {
```